examples/python/arkit_scenes/main.py

```diff
@@ -16,6 +16,15 @@
 from scipy.spatial.transform import Rotation as R
 from tqdm import tqdm
 
+DESCRIPTION = """
+# ARKitScenes
+This example visualizes the [ARKitScenes dataset](https://github.com/apple/ARKitScenes/) using Rerun. The dataset
+contains color images, depth images, the reconstructed mesh, and labeled bounding boxes around furniture.
+
+The full source code for this example is available
+[on GitHub](https://github.com/rerun-io/rerun/blob/latest/examples/python/arkit_scenes/main.py).
```

Comment:
I think these should link to the directory instead so users can see the long readme as well

Reply:
Good point